vacationtime-solution/Code.gs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// GLOBALS
+var APPROVAL_COLUMN = 9;
+var NOTIFIED_COLUMN = 10;
+var EMAIL_COLUMN = 2;
+var NAME_COLUMN = 3;
+var START_DATE = 6;
+var END_DATE = 7;
+var MANAGER_EMAIL = 4;
+    
+/** 
+* Creates menu items to run scripts.
```

Comment:
The purpose isn't to run scripts for the sake of running scripts. I'd change to:
/**
 * Add custom menu items when opening the sheet.
 */